test/attr.test.js

```diff
@@ -30,6 +31,22 @@ describe('#attr', () => {
       expect(wrapper.find('span')).to.not.have.attr('disabled')
     })
 
+    it('passes when attribute exists without a value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('itemscope')
+    })
+
+    it('passes when attribute exists with a falsey (but not false/null/undefined) value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('draggable')
+    })
+
+    it('passes when attribute exists but has value `false`', (wrapper) => {
+      expect(wrapper.find('article')).to.not.have.attr('hidden')
```

Comment:
I agree it's probably best to have a 1:1 behaviour with the FB codebase - for now this is already a huge improvement, so imo leave a note in the codebase and 🚢 unless you have the extra time to tackle this.